docs/power-payment-app.md

```diff
@@ -0,0 +1,127 @@
+# Power Payment App
+
+## Overview
+
+**What does it do?**
+
+PowerPay is a mobile payment application that allows users to send and receive money, top-up their accounts, and withdraw cash. It offers a flexible payment experience by utilizing both smartphones and physical devices called Choronko.
+
+**Who is using it?**
+
+- PowerPay Users: Individuals with smartphones who can initiate and manage transactions through the app.
+- PowerPay Kiosks: Physical kiosks that facilitate top-ups and withdrawals for users with smartphones and Choronko.
+- Choronko Users: Individuals who can send and receive payments using the Choronko device without needing a smartphone.
+
+## Technologies used
+
+- ReactJS for building the user interface and implementing a Progressive Web App (PWA) approach.
+- HTML and CSS for the structure and styling of the user interface.
+- RESTful APIs for communication with the backend server.
+
+## Architecture and Interaction
+
+### User Interface:
+
+- **Registration Process**: The registration process is a series of steps that allows users to create an account and access the payment app's features. The steps typically include:
+
+    1. **Step 1 - Personal Information**: Users enter their personal details, such as name, phone number etc. This information is necessary for account creation and verification.
+
+    2. **Step 2 - Pin Setup**: Users set up a secure pin to protect their account. The pin should meet specific criteria, such as minimum length of numbers etc.
+
+    3. **Step 3 - OTP Confirmation**: Once users have entered their information, set up a pin, they are provide with an OTP in which they now confirm their account for security measures.
+
+    4. **Step 4 - Confirmation**: Once users have entered their information, set up a pin, etc . they can proceed to complete the registration process. The "Register" button is clicked to finalize the registration.
+
+    **note:** **The above registration method is for a smartphone user.**
+    **See mockup:** ![reference image](/mockups_wireframes/Registration_process.png)
+
+
+- **Registartion process for a choronko user**:As a choronko user who wants to register, all you need to do is :
+
+1. **Step 1 - Initiating Registration**: The choronko user can initiate the registration process by calling the powerpay service and then providing  their personal information such as full name , phone number etc.
```

Comment:
He's not calling the PPService, he's sending an SMS to the API Gateway that will redirect it to our Translation service, which will send it to our PPService... but what does the Chronko user process doing here? I wanted you to discuss only the App. The choronko is not using the App.

Reply:
Please sir @stephane-segning 
 if that's the case, it implies the entire registration process for the choronko user should be removed right? 

becuase the choronko has nothing to do with the app during the registration process.

Reply:
Not only. Everything concerning the choronko user should be kicked out

Reply:
Undestood